doc/Importing_OVA_File.rst

```diff
@@ -166,6 +166,10 @@ If you are facing errors due to usage of old versions, perform the below mention
 
 #. Return to `CORTX on OVA <CORTX_on_Open_Virtual_Appliance.rst>`_, and resume following the instructions there.
 
+Click `here <https://kb.vmware.com/s/article/80467>`_ to navigate to the relevant page and resolve the below mentioned error.
```

Comment:
Why does it it say 'Click here to .... resolve the below mentioned error.  Will everyone see this error every single time?  Or only sometimes?  If everyone sees it every time, then put in the instructions to fix it.  If it only sometimes happen, then say, "If you see the below error, then . . . "

Reply:
I have asked @mukul-seagate11 and @sachitanands for updates for the same. I will update the same once I get clarity.